backend/crates/eiffelvis_core/src/domain/types.rs

```diff
@@ -27,16 +27,24 @@ pub struct BaseEvent {
     pub links: Vec<BaseLink>,
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize)]
+impl From<&BaseEvent> for BaseEvent {
+    fn from(ev: &BaseEvent) -> Self {
+        ev.clone()
```

Comment:
seems incorrect

Reply:
We were literally cloning all events on the HTTP GET api lol...